### DISCUSSION TITLE
Is there an easy way to port any PyTorch model into GGML?

### DISCUSSION THREAD

**Discussion by Connor56** [ggml-org/ggml#977] [General]
I want to run a BERT-like model on my CPU using GGML to see if it's faster than the PyTorch version. I can't find any documentation on how to use GGML. What's the easiest way of doing something like this?

**Comment by ggerganov** [comment]
Try this tutorial and see if it works for your model: https://github.com/ggerganov/llama.cpp/discussions/7712
